src/pricecypher/config_sections.py

```diff
@@ -0,0 +1,52 @@
+from pricecypher.endpoints import ConfigEndpoint
+from pricecypher.models import ConfigSection
+from pricecypher.rest import RestClient
+
+
+class ConfigSections(object):
+    """
+    Config API. Exposes all available operations on dataset configs, like fetching the available sections, and
+    retrieving the key-value pairs of a section and parsing it to a dictionary.
+
+    :param str bearer_token: Bearer token for PriceCypher (logical) API. Needs 'read:configuration' scope.
+    :param str config_base: (optional) Base URL for PriceCypher config service API.
+        (defaults to the static default_config_base, which by default is https://config.pricecypher.com)
+    :param RestClientOptions rest_options: (optional) Set any additional options for the REST client, e.g. rate-limit.
+        (defaults to None)
+    """
+
+    """ Default user-tool base URL """
+    default_config_base = 'https://config.pricecypher.com'
+
+    def __init__(self, bearer_token, dataset_id, config_base=None, rest_options=None):
```

Comment:
Out of curiosity, is type hinting generally not used in `__init__`? (I know the types are already in the docstring above)

Reply:
Type hinting is currently not used for any of the function parameters in this project.